index.bs

```diff
@@ -4322,12 +4332,15 @@ Note: The {{AuthenticatorTransport}} enumeration is deliberately not referenced,
 <div dfn-type="typedef" dfn-for="COSEAlgorithmIdentifier">
     A {{COSEAlgorithmIdentifier}}'s value is a number identifying a cryptographic algorithm.
     The algorithm identifiers SHOULD be values registered in the IANA COSE Algorithms registry [[!IANA-COSE-ALGS-REG]],
-    for instance, <code>-7</code> for "ES256" and <code>-257</code> for "RS256".
+    for instance, <code>-9</code> for "ESP256" and <code>-257</code> for "RS256".
 
     The COSE algorithms registry leaves degrees of freedom to be specified by other parameters in a [=COSE key=]. In order to promote interoperability, this specification makes the following additional guarantees of [=credential public keys=]:
         1. Keys with algorithm ES256 (-7) MUST specify P-256 (1) as the [=crv=] parameter and MUST NOT use the compressed point form.
+        1. Keys with algorithm ESP256 (-9) MUST NOT use the compressed point form.
         1. Keys with algorithm ES384 (-35) MUST specify P-384 (2) as the [=crv=] parameter and MUST NOT use the compressed point form.
+        1. Keys with algorithm ESP384 (-48) MUST NOT use the compressed point form.
         1. Keys with algorithm ES512 (-36) MUST specify P-521 (3) as the [=crv=] parameter and MUST NOT use the compressed point form.
+        1. Keys with algorithm ESP512 (-49) MUST NOT use the compressed point form.
```

Comment:
ECDSA with P521 with SHA512 is -36

Reply:
Thanks for catching the conflict! [Draft 12](https://www.ietf.org/archive/id/draft-ietf-jose-fully-specified-algorithms-12.html#name-elliptic-curve-digital-sign) changed the assignment request to -52. Fixed in c129a055c23e97b01caf513aa130a48d3904afe2.